### ISSUE TITLE
Return type of sign() is Decimal in d.ts 

### ISSUE THREAD

**Issue by gterras** [MikeMcl/decimal.js#209]
Hi, 

https://github.com/MikeMcl/decimal.js/blob/3dcadc7351eace5dd61a48ad5bfea8c1dd388ec2/decimal.global.d.ts#L289

But as per the docs : 

```javascript
.sign(x) ⇒ number
typeof Decimal.sign(new Decimal(1)) // "number"
```

**Comment by MikeMcl** [comment]
Corrected. Thank you.

**Comment by gterras** [comment]
Thanks a lot do you plan to tag a release soon ?

**Comment by MikeMcl** [comment]
Published *v10.4.2*.
